src/main/java/de/tum/in/www1/artemis/web/rest/dto/LtiLaunchRequestDTO.java

```diff
@@ -8,8 +8,16 @@
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
 public class LtiLaunchRequestDTO {
 
+    // Method and variable names need to match the LTI standard
+
     private String lis_person_sourcedid;
```

Comment:
Is this naming convention necessary for LTI to work?
Otherwise you could use lisPersonSourcedid and so on

Reply:
Indeed this is necessary for LTI to work since the request needs to be mapped to the DTO.